tests/models/wav2vec2/test_modeling_tf_wav2vec2.py

```diff
@@ -244,6 +250,33 @@ def check_ctc_loss(self, config, input_values, *args):
 
         self.parent.assertTrue(abs(labels.shape[0] * mean_loss - sum_loss) < 1e-2)
 
+    def check_seq_classifier_loss(self, loss, config, input_values, *args):
+        model = TFWav2Vec2ForSequenceClassification(config)
+
+        input_values = input_values[:3]
+        attention_mask = tf.ones(input_values.shape, dtype=tf.int32)
+
+        input_lengths = [input_values.shape[-1] // i for i in [4, 2, 1]]
+        labels = tf.random.uniform((input_values.shape[0],), maxval=len(model.config.id2label), dtype=tf.int32)
+
+        # pad input
+        for i in range(len(input_lengths)):
+            input_values[i, input_lengths[i] :] = 0.0
+            attention_mask[i, input_lengths[i] :] = 0
+
+        # disable dropout and batch normalization
+        tf.keras.backend.set_learning_phase(0)
+
+        masked_loss = model(input_values, attention_mask=attention_mask, labels=labels).loss.numpy().item()
+        unmasked_loss = model(input_values, labels=labels).loss.numpy().item()
+
+        # enable dropout and batch normalization
+        tf.keras.backend.set_learning_phase(1)
```

Comment:
`set_learning_phase` isn't wrong, but in the rest of the library we usually just pass a `training` bool to the models. Can you modify the layer to take that argument and pass it to the other layers, and then replace `set_learning_phase` here with `training=False`?